data/hu.kramo.Cartridges.metainfo.xml.in

```diff
@@ -16,6 +16,23 @@
   <url type="contribute">https://github.com/kra-mo/cartridges/blob/main/CONTRIBUTING.md</url>
   <developer_name translatable="no">kramo</developer_name>
   <launchable type="desktop-id">@APP_ID@.desktop</launchable>
+  <translation type="gettext">cartridges</translation>
+  <categories>
```

Comment:
I don't understand why these extra tags should be added. appstreamcli doesn't find any issues with the metainfo file currently and I would rather not duplicate strings.